LondonBritishLibrary/orient/BLorient801.xml

```diff
@@ -55,75 +110,74 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <extent>
                                     <measure unit="leaf">230</measure>
                                     <measure unit="blank">1</measure>
```

Comment:
could you include in a note the catalogue's information that the unnumbered blank leaf is at the end?